services/horizon/internal/ingest/processors/contract_data.go

```
@@ -114,10 +123,12 @@ func AssetFromContractData(ledgerEntry xdr.LedgerEntry, passphrase string) *xdr.
 		return nil
 	}
 	obj = assetIssuerEntry.Val.MustObj()
+	if obj == nil {
+		return nil
+	}
 	if obj.Type != xdr.ScObjectTypeScoBytes {
 		return nil
 	}
-	var err error
 	assetIssuer, err = strkey.Encode(strkey.VersionByteAccountID, obj.MustBin())
```

I think this is wrong : 
if objMjustBin() is nil ,then you shoudl return nil.
however, if `strkey.Encode` returns an error, you should return this error.